style-guide.md

```diff
@@ -129,15 +129,15 @@ Depending on the size of the company, the team description could also be the com
 
 DO: If it is product development a link to find out more about the product would be helpful if it's available.
 
-"The candidate will be working on our product development team on our [Analytics Product](http://google.com)"
+"The candidate will be working on our product development team on our [Analytics Product](http://google.com)" <!-- Possible NDA and/or company discretion conflicts from linking the product -->
 
```

Comment:
If there *is* an NDA, state that up front. And probably say *why* there needs to be an NDA. I am personally turned off by potential clients that want me to sign an NDA before they will even talk about details of a project.